Sources/MapboxNavigation/Style.swift

```diff
@@ -35,14 +35,14 @@ open class Style: NSObject {
     /**
      URL of the style to display on the map during turn-by-turn navigation.
      */
-    @objc open var mapStyleURL: URL = MapboxMaps.Style.navigationDayStyleURL
+    @objc open var mapStyleURL = URL(string: StyleURI.navigationDay.rawValue)!
```

Comment:
StyleURI is a Swift type that doesn’t bridge to Objective-C. This property must be available to Objective-C to participate in the UIAppearance system, which relies on Objective-C runtime reflection. The new code is uglier than the old code, but I don’t see a more elegant approach except for migrating the map SDK back to straight-up URLs.